include/odp/api/spec/schedule.h

```diff
@@ -187,6 +187,51 @@ void odp_schedule_prefetch(int num);
  */
 int odp_schedule_num_prio(void);
 
+/**
+ * Initialize schedule configuration options
+ *
+ * Initialize an odp_schedule_config_t to its default values.
+ *
+ * @param[out] config  Pointer to schedule configuration structure
+ */
+void odp_schedule_config_init(odp_schedule_config_t *config);
+
+/**
+ * Global schedule configuration
+ *
+ * Initialize and configure scheduler with global configuration options
+ * to schedule events across different schedule queues. The scheduler module
+ * should be started before creating any odp queues. This should happen before
+ * an application itself or any other ODP module creates any schedued queue.
+ * An application can pass NULL value to use default configuration.
```

Comment:
config must be called before any other schedule API function is called, except capability.

Typo: "schedued queue"

Reply:
Still needs update about "config must be called before any other sched call". Also has still the typo.

Here's my suggestion:
"This function must be called before scheduler is used (any other scheduler function is called except odp_schedule_capability() and odp_schedule_config_init()) or any queues are created (by application itself or by  other ODP modules)."

Reply:
ok